x-pack/plugins/maps/public/shared/components/multi_field_select.js

```diff
@@ -53,7 +53,7 @@ export function MultiFieldSelect({
 
 MultiFieldSelect.propTypes = {
   placeholder: PropTypes.string,
-  fields: PropTypes.object, // IndexedArray object
+  // fields: PropTypes.object, // IndexedArray object
```

Comment:
This should be either changed to PropTypes.oneOf or PropTypes.array

Reply:
just merged in changes from https://github.com/elastic/kibana/pull/34379